frontend/src/main/pages/GeneralEducation/search/GeneralEducationSearchPage

```diff
@@ -0,0 +1 @@
+console.log("GE search coming soon!");
```

Comment:
No.  Please make a proper placeholder along the lines of what you did in team02.  Further, this would not work anyway because the file has to end in .js.